README.md

```diff
@@ -1,25 +1,33 @@
 # Awesome List on 'Do No Harm' for Open Source [![Awesome Lists](https://srv-cdn.himpfen.io/badges/awesome-lists/awesomelists-flat.svg)](https://github.com/awesomelistsio/awesome)
 
-> A curated list of resources to help guide the implementation of "do no harm" principle in open source/science projects. 
+> A curated list of resources to help guide the implementation of "do no harm" principle in open source/science projects.
+
+Every piece of software has a negative impact on the world around us. From the lithium in your laptop and the electricity to power it, to the artists having their work stolen by image generators. Harm is unavoidable and sometimes even worth it, but we believe it's important that software practitioners properly deliberate the issue and aim to minimize harm done.
 
 ## Contents
 
-- [Section](#section)
-- [Another Section](#another-section)
+- [Large scale computing / AI / LLM](#large-scale-computing--ai--llm)
 
-## Section
+### Large scale computing / AI / LLM
 
-About this section. Optional. Keep this short and focus on the list.
+<details>
+  <summary>Data labelling</summary>
+  Data is manually labelled by humans [Jones, 2021]. The true cost of this activity is borne by marginalized people.
+  The work is carried out by humans living in poor countries, slums and refugee camps, the incarcerated [Wired, 2023] and the disabled [Zyskowski et al., 2025],
+  and involve exploitative practices such as poor pay, limited workers' rights, and unpaid labour [Jacobin, 2021], [Muldoon & Raekstad, 2022].
+  
+  **References**
 
-- [List item](http://example.com)
-- [List item](http://example.com)
+  [Jones, 2021] Work without the Worker: Labour in the age of platform capitalism
+  
+  [Wired, 2023] These Prisoners Are Training AI
```

Comment:
Any preferences for how to cite? Especially regarding access date and archives / permalinks for digital material